pcu/PCU.h

```diff
@@ -130,6 +125,25 @@ void Protect() noexcept;
 /*MPI_Wtime() equivalent*/
 [[nodiscard]] double Time() noexcept;
 
+/**
+ * @brief Initialize the underlying parallel library.
+ *
+ * This may be MPI (or a stub, given SCOREC_NO_MPI). This function abstracts
+ * the difference.
+ *
+ * @return 0 on success, nonzero otherwise.
+ */
+int PCU_Init(int *argc, char ***argv);
+/**
+ * @brief Finalize the underlying parallel library.
+ *
+ * This may be MPI (or a stub, given SCOREC_NO_MPI). This function abstracts
+ * the difference.
+ *
+ * @return 0 on success, nonzero otherwise.
+ */
+int PCU_Finalize();
+
 PCU* PCU_GetGlobal();
```

Comment:
I know this isn't something you added, but can we remove this? I don't think we store a global PCU pointer anymore.

Reply:
Sure, it looks like there is no definition anywhere for the C++ or C api version of this call so I will remove it.